keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterSpec.kt

```diff
@@ -51,23 +55,45 @@ data class TitusClusterSpec(
   private val _defaults: TitusServerGroupSpec,
   @JsonInclude(JsonInclude.Include.NON_EMPTY)
   val overrides: Map<String, TitusServerGroupSpec> = emptyMap(),
-  @JsonIgnore
-  override val deliveryArtifact: DeliveryArtifact? = null,
-  @JsonIgnore
+  // The following 3 properties are write-only as they're filled by resolvers, but not output in JSON
+  @JsonProperty(access = Access.WRITE_ONLY)
+  override val artifactType: ArtifactType? = ArtifactType.docker,
```

Comment:
Doesn't making them `WRITE_ONLY` imply we're expecting users to be able to set them? They'll be exposed in the API schema, right? (Same comment applies to a couple places this is used).

Reply:
Hmm... So I went down this path originally because the API doc compat tests started failing and I thought when I added `@JsonIgnore` somehow that prevented the resolvers from setting the property, but now that I'm writing this I realize that doesn't make any sense. It was probably because I had also moved the property declarations out of the constructor and into the class body at that point. Let me take another look and fix if possible.

Reply:
Replaced annotations with `@JsonIgnore`.